src/common/io.py

```diff
@@ -30,7 +30,12 @@
 import numpy as np
 import scipy.io
 from scipy import interpolate
-from scipy.io.matlab.mio4 import MatFile4Reader, VarReader4, convert_dtypes, mdtypes_template, mxSPARSE_CLASS
+
+from pkg_resources import parse_version
```

Comment:
I don't think we should use `pkg_resources` because then we are introducing dependency on another python package and we would like to avoid that. Only alternate solution I see is to verify that each element from `scipy.__version__.split('.')`

Reply:
ok

Reply:
Looks good!